Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.mm

```diff
@@ -571,6 +573,24 @@ static CGFloat effectivePointsPerMeter(CALayer *caLayer)
     completionHandler();
 }
 
+void ModelProcessModelPlayerProxy::setEnvironmentMap(Ref<WebCore::SharedBuffer>&& data)
+{
+    m_environmentMapData = WTFMove(data);
+    if (m_modelRKEntity)
+        applyEnvironmentMapDataAndRelease();
```

Comment:
We need to handle the removal case as mentioned earlier.

Reply:
Added.